packages/react-devtools-extensions/src/main.js

```diff
@@ -245,8 +245,8 @@ function createPanelIfReactLoaded() {
           }
         };
 
-        const viewSourceLineFunction = (url, line) => {
-          chrome.devtools.panels.openResource(url, line);
+        const viewUrlSourceFunction = (url, line, col) => {
+          chrome.devtools.panels.openResource(url, line, col);
```

Comment:
Wait, are you sure this works? The API docs says the 3rd parameter is callback, not column https://developer.chrome.com/docs/extensions/reference/devtools_panels/#method-openResource